src/protagonist/Engine.Tests/Ingest/Image/ImageServer/Clients/CantaloupeThumbsClientTests.cs

```diff
@@ -96,4 +96,30 @@ public async Task GenerateThumbnails_ReturnsNothing_WhenCantaloupeReturns400()
         // Assert
         thumbs.Should().HaveCount(0);
     }
+    
+    [Fact]
+    public async Task GenerateThumbnails_Ignores400_AndProcessesRest()
+    {
+        // Arrange
+        var assetId = new AssetId(2, 1, nameof(GenerateThumbnails_ReturnsNothing_WhenCantaloupeReturns400));
+        var context = IngestionContextFactory.GetIngestionContext(assetId: assetId.ToString());
+
+        var thumbSizes = new List<string> { "!1024,1024", "!400,400" };
+        
+        // first size is 400, then 200 after
```

Comment:
is this correct? the `thumbSizes` looks like 1024, then 400

Reply:
Will clarify, it's statusCodes.